### PR TITLE
Build & Doc: simplify install instructions

### PR DESCRIPTION
**Pull request by rht**
This way, explicit support for more systems can be added incrementally.

> * We run Python 3.5 internally, so the Python 3.5 variants will be much more thoroughly performance tested. Please let us know if you see any issues on 2.7.

Note that the tests for python 2.7 can be readily added to the travis.yml.

For OSX, the supported version range should be made explicit (homebrew recommends at least 10.10).

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -13,6 +13,24 @@ build:
 	docker build -t quay.io/openai/universe .
 	docker build -f test.dockerfile -t quay.io/openai/universe:test .
 
+deps:
+	UNAME_S := $(shell uname -s)
```
Comment (tlbtlbtlb) [inline Makefile]: Syntax error here.

Comment (rht) [inline Makefile]: (right, sorry I realized that only after)

**Review by tlbtlbtlb — CHANGES_REQUESTED** [review]
People have reported installing universe following the instructions without issues on Linux Mint and RHEL. And with a little fiddling, on Windows/Bash and OSX with MacPorts. This seems to make it worse for them: `make deps` will silently do nothing.
At least, `make deps` should have else clauses that print the current instructions.

Also, our Travis flow is to create a container that runs Tox. I'd rather have the `make deps` run inside that. It should get the same results as running tox in the universe directory. See tox.ini there.

**Comment by rht** [comment]
> People have reported installing universe following the instructions without issues on Linux Mint and RHEL. And with a little fiddling, on Windows/Bash and OSX with MacPorts. This seems to make it worse for them: make deps will silently do nothing. At least, make deps should have else clauses that print the current instructions.

Noted. I have added a more informative message.

- [ ] Except for OSX case where it still defaults to `brew`.
- [ ] `make deps` in tox

**Comment by rht** [comment]
@tlbtlbtlb 

Update: `make deps` for 14.04 has been tested through travis. By extrapolation, 16.04 should be fine too. The Darwin part is untested, however.

Also, I just noticed that the test suite does run a Python 2.7 virtualenv (which is still not sufficient to detect any lingering performance regression).

**Comment by Baschdl** [comment]
@rht Couldn't you use travis to test also 16.04 and Mac OS automatically?

**Comment by rht** [comment]
@Baschdl that will certainly do but might be overkill to test-run an already dockerized env, but can be added in a test-commit to just to test whether the build script runs well in 16.04 and macOS.

**Pull request closed without merging**